server/odc-service/src/main/java/com/oceanbase/odc/service/queryprofile/helper/CustomStringSubstitutor.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.queryprofile.helper;
+
+import java.util.Map;
+
+/**
+ * @author liuyizhuo.lyz
+ * @date 2024/4/12
+ */
+public class CustomStringSubstitutor {
```

Comment:
this name is too common, how about renaming it to tell the user this substitutor is only used for query profiling~

Reply:
done, renamed into `PlanParameterSubstitutor`